### PR TITLE
Add discussion on tabs in screen

### PR DESCRIPTION
**Pull request by lbel**

### CODE REVIEW

**Comment by betatim** [comment]
Is this beginner stuff? I think so ... however it seems the rate of adopting `screen` was very low in the starterkit. Should we add this here or in`second-analysis-steps`. Either way we need to ponder how we can show people that this is useful stuff and not just emacs like keybindings.

File: 02-screen.md [hunk 1]
```diff
@@ -134,3 +134,32 @@ $ kinit
...
+>
+> Screen supports some features beyond detaching a session. A very useful feature is different sessions in tab pages, all within a single instance of `screen`. 
+> In order to maximise the usefulness of this feature, you need to set the screen status bar. The simplest way to do this is by appending the following line to the file `~/.screenrc` (create it if it doesn't already exist):
+> ```
+> # Status lines
+> hardstatus off
+> hardstatus alwayslastline
+> hardstatus string '%{= BG}%{.g}[ %{.G}%H %{.g}][ %{= Bw}%?%-Lw%?%{Yr}%{.k}%n*%f%? %t%?%?(%u)%?%{.r}%{Bw}%?%+Lw%? %{.g}%=][%{.W} %Y-%m-%d %c %{.g}]'
+> ```
+> Note that this has predefined colours and layout that you can easily change yourself, for a detailed discussion see eg. http://sourceopen.com/2014/06/09/gnu-screen-status-bar-tips-tricks-basics/.
```
Comment (betatim) [inline 02-screen.md]: can you turn the link into a link with a useful name?

Comment (lbel) [inline 02-screen.md]: Done!

**Comment by betatim** [comment]
I am not sure this is beginner stuff or not, nobody else seems to have an opinion either so this is being merged! (can't believe there is no emoji for that :cry:)

The starterkit sends you some (virtual) :pizza:

**Pull request merged**